bitnami/postgresql-ha/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: postgresql-ha
-version: 1.4.3
+version: 1.4.4
```

Comment:
Please bump (at least) minor version since the structure of the **values.yaml** is affected.

Reply:
Also, if we are not updating the major, I would add a backwards compatibility feature